testing/integration/ess/monitoring_probe_preserve_text_cfg_test.go

```diff
@@ -173,12 +178,21 @@ func (runner *MonitoringTextRunner) TestMonitoringLiveness() {
 	require.NoError(runner.T(), err)
 	require.Equal(runner.T(), http.StatusOK, statusCode, "non-200 status code; got response: %s", string(overrideResp))
 
+	// verify the new policy revision was applied
+	newPolicyRevision := agent.PolicyRevision + 1
+	require.Eventually(
+		runner.T(),
+		tools.IsPolicyRevision(ctx, runner.T(), runner.info.KibanaClient, runner.policyID, newPolicyRevision),
```

Comment:
```suggestion
		tools.IsPolicyRevision(ctx, runner.T(), runner.info.KibanaClient, runner.agentID, newPolicyRevision),
```

Reply:
```
=== RUN   TestMonitoringPreserveTextConfig/TestMonitoringLiveness
    monitoring_probe_preserve_text_cfg_test.go:214: component state: Healthy: communicating with pid '65596'
    monitoring_probe_preserve_text_cfg_test.go:214: component state: Healthy: communicating with pid '65577'
    monitoring_probe_preserve_text_cfg_test.go:214: component state: Healthy: communicating with pid '65556'
    monitoring_probe_preserve_text_cfg_test.go:214: component state: Healthy: communicating with pid '65629'
    monitoring_probe_preserve_text_cfg_test.go:214: component state: Healthy: communicating with pid '65651'
    tools.go:31: failed to get agent document to check policy revision: Agent 3e6a783e-0e97-4ad3-8674-fc99eeb2509a not found
```

Reply:
Passed the wrong id, it's fixed now. I've also switched to using a higher-level method on the Kibana client, which lets me wait for the exact policy revision we expect.